Prac_05/word_occurances.py

```diff
@@ -0,0 +1,18 @@
+individual_different_words = {}
+input_text = input("Text: ").lower()
+words = input_text.split()
+# print(words)
+for word in words:
+    frequency = individual_different_words.get(word,0)
```

Comment:
PyCharm would have warned you about formatting on this line. Watch for these and fix them.